app/code/Magento/Catalog/Block/Product/View/Details.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+/**
+ * Product details block
+ * Holds a group of blocks to show as tabs
+ *
+ * @author     Magento Core Team <core@magentocommerce.com>
+ */
+
+namespace Magento\Catalog\Block\Product\View;
+
+/**
+ * @api
```

Comment:
Could you provide more details about the necessity to mark this class with `@api` annotation, please?

Reply:
@rogyar , I've added it because the Block classes (e.g. `Magento\Catalog\Block\Product\View`) on the same level has the same annotation. So far, I couldn't understand why they are marked so. Do you know something on that ?